css/index.css

```diff
@@ -112,6 +117,36 @@ p {
     justify-content: center;
 }
 
```

Comment:
Nice job letting the parent .box take care of all the styling of the individual boxes and only changing the colors.